configure.ac

```diff
@@ -3,4 +3,27 @@ AM_INIT_AUTOMAKE(ke-recv-extra, 0.2)
 AM_CONFIG_HEADER(config.h)
 AC_PROG_CC
 
+PKG_CHECK_MODULES(OSSO, libosso)
+AC_SUBST(OSSO_CFLAGS)
+AC_SUBST(OSSO_LIBS)
+
+PKG_CHECK_MODULES(GLIB, glib-2.0)
+AC_SUBST(GLIB_CFLAGS)
+AC_SUBST(GLIB_LIBS)
+
+PKG_CHECK_MODULES(GCONF, gconf-2.0)
+AC_SUBST(GCONF_FLAGS)
+AC_SUBST(GCONF_LIBS)
+
+PKG_CHECK_MODULES(DBUS, dbus-1)
+AC_SUBST(DBUS_CFLAGS)
+AC_SUBST(DBUS_LIBS)
+
+PKG_CHECK_MODULES(DBUS_GLIB, dbus-glib-1)
+AC_SUBST(DBUS_GLIB_CFLAGS)
+AC_SUBST(DBUS_GLIB_LIBS)
+
+LOCALEDIR=`pkg-config --variable=localedir osso-af-settings`
+AC_SUBST(LOCALEDIR)
+
```

Comment:
This can be done in Makefile.am

Reply:
I think we do this on configure.ac in most of our repos, don't we?

Reply:
https://github.com/maemo-leste/ke-recv/blob/30f2486b1aa66c5c0b02d0884e3a2754ff39ed73/src/Makefile.am#L14

Reply:
configure.ac is the right place for it, not Makefile.am

Reply:
Agree. Will do the same in ke-recv.

Reply:
Please check commit  https://github.com/maemo-leste/ke-recv/commit/9c73ad50a8e37d87bfdd20be1664e8a4a3cb4041
Using AC_DEFINE_UNQUOTED macro seems to be more appropriate here.